test/test_ops.py

```diff
@@ -648,7 +651,7 @@ def test_noncontiguous_samples(self, device, dtype, op):
     # Cases test here:
     #   - out= with the correct dtype and device, but the wrong shape
     @ops(_ops_and_refs, dtypes=OpDTypes.none)
-    @skipIfTorchInductor("Inductor does not support complex dtype yet")
+    @patch.object(torch, "manual_seed", deterministic_torch_manual_seed)
```

Comment:
The test has now been marked as skipped in master. So we can remove the patch using `deterministic_torch_manual_seed`